Makefile

```diff
@@ -0,0 +1,7 @@
+lint:
+	@echo ? Running golangci-lint
+	@docker-compose -f tools/golang/docker-compose.yml up lint
```

Comment:
I'm not sure what the `?` in the echo call is supposed to do (except print a <kbd>?</kbd>?) but it's maybe irrelevant because it looks like all you're doing is printing what's about to happen. Which is exactly what `make` would do by default if you got rid of your `echo`s and didn't preface your steps with `@`.

Reply:
Yeah the ? was more of just breaking out the stdout from the target and the start. 

```COSML-1547398:trafficcontrol mhoppa509$ make lint
? Running golangci-lint
Starting golang_lint_1 ... done
Attaching to golang_lint_1
```

vs 

```
COSML-1547398:trafficcontrol mhoppa509$ make lint
docker-compose -f tools/golang/docker-compose.yml up lint
Starting golang_lint_1 ... 
```
And yes if you omit the @ it would print out the steps but its verbatim instead of something as `Running golangci-lint` but I can change it if you are really against it.

TBH I am not very picky on Makefile format.

Reply:
I personally think the default behavior of printing exactly what it's doing is best, but I see what you're getting at. If that's necessary or even just beneficial for CI reasons, I'd suggest just pre-pending the `?` to the command that's actually being run rather than abbreviating it. Something like:

```make
lint:
	@echo -n '? '
	docker-compose -f tools/golang/docker-compose.yml up lint
```
which will output
```console
$ make
? docker-compose -f tools/golang/docker-compose.yml up lint
Starting golang_lint_1 ... done
Attaching to golang_lint_1
...
```
It's _possible_ that'll still mess with tools that invoke `make` and parse the output, but probably not likely. And also probably not even a big deal if it does either.